core/message-tags-3.3.md

```diff
@@ -80,6 +89,24 @@ Individual tag keys MUST only be used a maximum of once per message. Clients
 receiving messages with more than one occurrence of a tag key SHOULD discard all
 but the final occurrence.
 
+### Empty tagged messages
+
+Servers MUST accept `PRIVMSG` and `NOTICE` events sent with client-only tags even if
```

Comment:
Alternative idea to consider: use an empty PRIVMSG instead (`@whatever PRIVMSG #chan :`) which also doesn't require syntax changes, but doesn't require new verb.